### PR TITLE
Verify contracts asynchronously from web / api pod on indexer pod

### PR DESCRIPTION
**Pull request by rkachowski**
### Description

Uses cluster communication to submit a smart contract for verification from web / api pod against an indexer pod. This requires the indexer to listen for contract submission events and to perform the actual verification. 

Due to this being an async operation a message is now displayed after the submission to inform the user that this is happening. This assumes that replication lag is low enough that the contract will be available in the replica db by the time the "View Contract" button has been clicked - given current stats this seems to be a reasonable assumption, but this approach may have to change in future.
 
 ### Other changes

* Set `application_name` on replica1 repo instance so that it's connections appear in `pg_stat_activity` with the correct name

### Tested

* deployed to rc1staging and saw verified contracts + log messages referencing the insertion

> This is actually pretty difficult to test due to the async + distributed nature and the UI is essentially a fallback for when no api routes exist. Essentially, upon entering a contract address within the text field an ajax call is immediately made to `/api/?module=contract&action=getsourcecode&address=$CONTRACT_ADDRESS`. This results in the system checking sourcify for an existing contract, and if found, directly making the request to publish the contract within blockscout db.

> Because of this, the webpage directly calls a blockscout api pod instance which then uses cluster communication to ask the indexer to insert the contract. This happens outrageously quickly and ends up showing that the contract is already verified via a field verification message. Before this feature was implemented the api pod had no means of writing to the db and the request would fail, in this case the web pod will be the component which sends the the request to the indexer.

> tl;dr - verification but (unexpectedly) faster

### Issues

 - Fixes https://github.com/celo-org/data-services/issues/443

### CODE REVIEW

File: apps/explorer/lib/explorer/repo.ex [hunk 1]
```diff
@@ -153,5 +153,18 @@ defmodule Explorer.Repo do
       otp_app: :explorer,
       adapter: Ecto.Adapters.Postgres,
       read_only: true
+
+    alias Explorer.Repo, as: ExplorerRepo
+
+    def init(_, opts) do
+      extra_postgres_parameters = [application_name: ExplorerRepo.get_application_name() <> "-replica1"]
```
Comment (carterqw2) [inline apps/explorer/lib/explorer/repo.ex:160]: What's `replica1`?

Comment (rkachowski) [reply apps/explorer/lib/explorer/repo.ex:160]: it's the name of the repo defined in this module, added upstream that is intended to be connected to a read replica.

https://github.com/celo-org/blockscout/pull/746/files/6c4c7a59f50d4169b191293c11e3206eeb0aa0ca#diff-4b8ee0e9de96fe0baf675b2d78397e1862b9a61d46a2d7b8ea6c3d67b7190031R151

**Review by carterqw2 — APPROVED** [review]

File: apps/explorer/lib/explorer/repo.ex (hunk 1 above)
Comment (carterqw2) [reply apps/explorer/lib/explorer/repo.ex:160]: Gotcha.
